### DISCUSSION TITLE
v4.9

### DISCUSSION THREAD

**Discussion by Antynea** [Antynea/grub-btrfs#149] [General]
### Improvement:

* grub-btrfs.path:
  * Allows the unit to start and stop automatically when the mount point is detected.  
If the mount point is already mounted when the unit is activated `systemctl enable grub-btrfs.path`,  
it will be necessary to start the unit for monitoring to begin `systemctl start grub-btrfs.path`.  
Refer #139 and https://github.com/Antynea/grub-btrfs/issues/138#issuecomment-769764290 for further explanations.  
#### Remember to modify your configuration accordingly.  

#
### Miscellaneous:
* Makefile:
  * Make sure make is invoked with appropriate permissions (#146)

* Readme :
  * Update "Automatically update grub" section following the recent change in the grub-btrfs.path unit.
  * Add information to be able to update the grub-btrfs menu every time the system is restart/shutdown.

<hr /><em>This discussion was created from the release <a href='https://github.com/Antynea/grub-btrfs/releases/tag/v4.9'>v4.9</a>.</em>

**Comment by Antynea** [comment]
I think the time has come.
@maximbaz , If you have some time, could you releases this version on Arch linux please?

You can discuss the version here.

**Reply by max-baz** [reply]
It's pushed 🙂 Good job!
